aspnet/publishing/iis.rst

```diff
@@ -54,6 +54,27 @@ For more information on the ASP.NET Core Module, including configuration of the
 Application configuration
 -------------------------
 
+Enabling the `IISIntegration` service
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Ensure that the `UseIISIntegration()` is injected:
+
+.. code-block:: csharp
+
+  var host = new WebHostBuilder()
+    .UseKestrel()
+    .UseIISIntegration()
+    .UseStartup<Startup>()
+    .Build();
```

Comment:
Going through the tutorial/getting started I can't see the UseContentRoot part: https://docs.asp.net/en/latest/getting-started.html

Should the getting started get updated as well or leave this as is?